src/mmpanelbase.cpp

```diff
@@ -43,6 +43,18 @@ mmListCtrl::~mmListCtrl()
 {
     if (attr1_) delete attr1_;
     if (attr2_) delete attr2_;
+    /*
```

Comment:
Here should be m_col_width validation. If the variable isemptyor not like *%d* skip save settings.